src/main/java/com/cinema_booking/booking/exception/RoomNotFindException.java

```diff
@@ -0,0 +1,8 @@
+package com.cinema_booking.booking.exception;
+
+public class RoomNotFindException extends RuntimeException {
+
+    public RoomNotFindException() {
```

Comment:
You could add some parameter here to give some details about it

Reply:
added